pkg/prove/proof_test.go

```diff
@@ -88,7 +88,7 @@ func TestTxSharePosition(t *testing.T) {
+			txs:  manySmallTxsWithoutRandomness,
```

Comment:
[nit][optional]

descriptive comment for future reference of why this test is needed.